packages/labs/router/src/routes.ts

```diff
@@ -85,6 +85,11 @@ export class Routes implements ReactiveController {
    */
   routes: Array<RouteConfig> = [];
 
+  /**
+   * A fallback route that is always matched after {@link routes}.
+   */
+  private readonly fallbackRoute?: RouteConfig;
```

Comment:
I think this should have the same visibility as `routes`

Reply:
Done!